instrumentation/kamon-elasticsearch/src/main/resources/reference.conf

```diff
@@ -0,0 +1,23 @@
+# ================================================== #
+# kamon Elasticsearch client reference configuration #
+# ================================================== #
+
+kamon.instrumentation.elasticsearch {
+}
+
+kanela {
+  modules {
+    elasticsearch-driver {
+
+      name = "Elasticsearch client"
+      description = "Elasticsearch Client Instrumentation"
+      instrumentations = [
+        "kamon.instrumentation.elasticsearch.ESInstrumentation"
+      ]
+
+      within = [
+        "org.elasticsearch..*"
```

Comment:
Can this be narrowed down?

Reply:
Narrowed to "org.elasticseearch.client..*"